content/projects/nodejs/sql/_index.md

```diff
@@ -1,8 +1,10 @@
 ---
 title: Node & SQL assignment
 ready: true
-weight: 2
-pre: "<b>2 (alt): </b>"
+prerequisites:
+  hard: ["workshops/node-and-sql"]
```

Comment:
another prerequisite is projects/sql ( i think).

There is a project tht is pure sql. We learn sql, then we learn how o access sql from node.